docker-storage-setup.1

```diff
@@ -98,18 +98,21 @@ WIPE_SIGNATURES:
       overwritten as suitable. This also means that if there is any
       data on disk, it will be lost.
 
-DOCKER_ROOT_VOLUME:
+ROOT_VOLUME_MOUNT_DIR:
       By default no new volume and filesystem will be setup
```

Comment:
@shishir-a412ed Can we improve the documentation of this option. Something like as follows.

`Create a logical volume and mount it on specified directory path. By default no new logical volume will be created.` 

I guess this itself is enough to begin with. Short and sweet.